ui/desktop/src/goosed.ts

```diff
@@ -22,7 +22,7 @@ export const findAvailablePort = (): Promise<number> => {
 // Function to fetch agent version from the server
 const fetchAgentVersion = async (port: number): Promise<string> => {
   try {
-    const response = await fetch(`http://127.0.0.1:${port}/api/agent/versions`);
+    const response = await fetch(`http://127.0.0.1:${port}/agent/versions`);
```

Comment:
I don't think we really need this function as it is only in the menu now

Reply:
+1 but leaving it for now in case we revisit